hebrew_names/__init__.py

```diff
@@ -0,0 +1,69 @@
+from os.path import abspath, join, dirname
+import codecs
+import random
+
+
+__title__ = 'hebrew_names'
+__version__ = '0.0.1'
+__author__ = 'Aluma Gelbard'
+__license__ = 'MIT'
+
+
+full_path = lambda filename: abspath(join(dirname(__file__), filename))
+
+
+FILES = {
+    'first:jew:male': full_path('dist.jew.male.first'),
```

Comment:
Maybe this could also be generates from the Enum so adding new ethnicities becomes easier

Reply:
generated with constants lists

done